data/json/recipes/recipe_armor.json

```diff
@@ -8441,6 +8487,61 @@
   ]
 },{
   "type" : "recipe",
+  "result": "armor_scavenger",
+  "category": "CC_ARMOR",
+  "subcategory": "CSC_ARMOR_SUIT",
+  "skill_used": "tailor",
+  "difficulty": 8,
+  "time": 80000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities" : [
+    {"id":"CUT","level":1,"amount":1}
+  ],
+  "tools": [
+    [
+      [ "soldering_iron", 18 ],
+      [ "toolset", 18 ]
+    ],
+    [
+      [ "needle_bone", 90 ],
+      [ "needle_wood", 90 ],
+      [ "sewing_kit", 90 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "jacket_flannel", 1 ],
+      [ "jacket_army", 1 ]
```

Comment:
Hmm. There's also the army pants / ski pants / motorcycle pants, think if I remove the flannel jacket, then that should also be just army pants.